src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -50,7 +50,7 @@ interface ButtonGroupProps extends VibeComponentProps {
   tooltipMoveBy?: MoveBy;
 }
 
-const ButtonGroup: React.ForwardRefExoticComponent<ButtonGroupProps & React.PropsWithChildren<unknown>> & {
+const ButtonGroup: VibeComponent<ButtonGroupProps, HTMLDivElement> & {
```

Comment:
Insert `children` to `ButtonGroupProps` or extend the `ButtonGroupProps` interface with `PropsWithChildren`:
```
interface ButtonGroupProps extends VibeComponentProps {
  ...same props as today,
  children?: React.ReactNode
}
```

or better I think:
```
interface ButtonGroupProps extends VibeComponentProps, React.PropsWithChildren {
```